ZXY6005S.py

```diff
@@ -0,0 +1,159 @@
+# This file enables communication with ZXY-6005S Power Supply Units
+# Documentation for Serial Commands: https://docs.google.com/document/d/1sobaDhIX-cFmzpiBEMtt-2S7j5h_ogMujiykYV2Dyv8/edit?usp=sharing
+# Data Sheet for ZXY-6005S Power Supply Unit https://drive.google.com/drive/u/0/folders/1QRbtmRMBnCshlBtOnuKX8HSpySGEhDeM
+# Author: Teresa LaBolle
+
+import serial
+import serial.tools.list_ports
+from enum import Enum
+
+class Commands(Enum):
+    '''Get power supply unit model. Response: ZXY-6005S'''
+    MODEL = 'a'
+    '''Get firmware version. Response: R2.7Z'''
+    FIRMWARE_VERSION = 'v'
+    '''Set amp hour counter to specified value. Response: Asa(value)'''
+    SET_AMP_HOUR = 'sa'
+    '''Return the amp hour reading. Response: Ara(5 digit value)'''
+    RETURN_AMP_HOUR = 'ra'
+    '''Set voltage to specified value. Response: Asu(value)'''
+    SET_VOLTAGE = 'su'
+    '''Return voltage measurement. Response: Aru(5 digit value)'''
+    RETURN_VOLTAGE = 'ru'
+    '''Set current limit to specified value. Response: Asi(value)'''
+    SET_CURRENT_LIMIT = 'si'
+    '''Return current in amps. Response: Ari(4 digit value)'''
+    RETURN_CURRENT = 'ri'
+    '''Return Mode [Constant Voltage(CV) or Constant Current(CC)] Response: Arc0 or Arc1'''
+    RETURN_MODE = 'rc'
+    '''Return temperature in Celsius. Response: Art(3 digit value)'''
+    RETURN_TEMP = 'rt'
+    '''Set power output (On/OFF). Response: Aso(1 OR 0)'''
+    SET_OUTPUT = 'so'
+
+
+class ZXY6005S:
+    BAUDRATE = 9600
+    INPUT_DELAY = 0.001
+    BYTESIZE = serial.EIGHTBITS
+    PARITY = serial.PARITY_NONE
+    STOPBITS = serial.STOPBITS_ONE
+    TIMEOUT = 1
+
+    def __init__(self):
+        '''construct objects, using location for X, Y, and Z power supplies'''
```

Comment:
It is also recommended that parameters such as device location/name is a configurable item for the constructor as this could change with different supplies or different environment configurations. It is also acceptable to add defaults to the initializers if they are generic settings that may be the same across environments.

Example:

```python3
def __init__(self, dev_1: str = '/dev/TTYUSBS10'):
    ...
```